src/test/java/org/gridsuite/shortcircuit/server/entities/AnalysisParametersEntityTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.entities;
+
+import com.powsybl.shortcircuit.ShortCircuitConstants;
+import org.assertj.core.api.WithAssertions;
+import org.gridsuite.shortcircuit.server.dto.ShortCircuitPredefinedConfiguration;
+import org.junit.jupiter.api.Test;
+
+class AnalysisParametersEntityTest implements WithAssertions {
+    @Test
+    void testUpdateSimple() {
+        AnalysisParametersEntity entity1 = new AnalysisParametersEntity();
+        AnalysisParametersEntity entity2 = new AnalysisParametersEntity().setMinVoltageDropProportionalThreshold(Double.MAX_VALUE);
+        assertThat(entity1).as("verification").usingRecursiveComparison().isNotEqualTo(entity2);
+        entity1.updateWith(entity2);
+        assertThat(entity1).as("check").usingRecursiveComparison().isEqualTo(entity2);
+    }
+
+    @Test
+    void testUpdate2() {
+        AnalysisParametersEntity entity1 = new AnalysisParametersEntity();
+        AnalysisParametersEntity entity2 = new AnalysisParametersEntity(
+                ShortCircuitConstants.DEFAULT_WITH_LIMIT_VIOLATIONS,
```

Comment:
Why don't you use these ShortCircuitConstants in the definition of default values in the entity ?

Reply:
Because their the defaults of PowSyBl, not the GridSuite ones.  
And I use this fact to create differences in this test.